publisher/src/main/java/com/pavlovmedia/oss/jaxrs/publisher/impl/swagger/SwaggerEndpoint.java

```diff
@@ -101,10 +101,14 @@ protected void deactivate() {
     
     @Override
     public Set<Class<?>> classesFromContext(final Application app, final ServletConfig sc) {
-        return publisher.getRawEndpoints().stream()
+        HashSet<Class<?>> ret = new HashSet<>();
+        publisher.getRawEndpoints().stream()
                 .map(o -> o.getClass())
                 .filter(c -> c.isAnnotationPresent(Api.class))
-                .collect(Collectors.toSet());
+                .forEach(ret::add);
+        publisher.getReaderListeners().forEach(ret::add);
+        ret.forEach(System.out::println);
```

Comment:
This looks like a debug print.  Don't you want to use logger.log(LogService.LOG_DEBUG) here or leave it commented out?